redocly.yaml

```diff
@@ -281,6 +281,11 @@ openapi:
   excludeFromSearch: true
 
 search:
-  ai:
-    hide: false
   engine: typesense
+
+aiAssistant:
+  hide: false
+  trigger:
+    hide: false
+    inputType: button
+    inputIcon: redocly
```

Comment:
what is the zero-config? is any of this actually needed?

Reply:
I think the only needed config is the `inputIcon` because default is `sparkles`.

```suggestion
  trigger:
    inputIcon: redocly
```

Reply:
Let's go with the sparkles... let's use the defaults ourselves.